languages/python/exercises/concept/conditionals/.docs/after.md

```diff
@@ -0,0 +1,137 @@
+## General
+
+Uptil now most program examples seen in the concept exercises ran in a sequential manner. Where code executed on after the other. However, this doesn't allow for implementation of more complex logic. Conditional statements allows program to execute different lines of code based on some factors. Python uses `if/else` statements to achieve this.
```

Comment:
The `after.md` document should be consider separate from the exercise, as in we'll have students access this document at any point in time, including later on when they'll likely have forgotten the exact context of the exercise. As such, the after.md document should not refer to any exercises.